commons/src/main/java/io/hyscale/commons/utils/ResourceSelectorUtil.java

```diff
@@ -18,10 +18,16 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import io.hyscale.commons.models.ResourceFieldSelectorKey;
 import io.hyscale.commons.models.ResourceLabelKey;
+import io.kubernetes.client.openapi.models.V1Pod;
 
 public class ResourceSelectorUtil {
+	private static final String DOT=".";
+	private static final String NAMESPACE="namespace";
```

Comment:
For field selector use ResourceFieldSelectorKey to implement a new enum which just deals with the k8s Fields
may be something like MetadataFieldSelector/ ResourceFieldSelector